README.md

```diff
@@ -7,6 +7,33 @@ This repo contains 4 jvm packages:
 * [dids](./dids) - did generation and resolution
 * [credentials](./credentials) - creation and verification of verifiable claims
 
+# Quickstart
```

Comment:
eventually we can have the offical link pointing to the latest release

Reply:
Yes!